src/components/PageSections/Menu/ThemesTopicsMenu.js

```diff
@@ -1,63 +1,173 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { injectIntl } from 'react-intl';
 import PropTypes from 'prop-types';
 import { misc as i18n2 } from 'js/i18n/definitions';
+import { useMobileQuery } from 'js/helpers/reactMediaQueries.js';
+import ExternalLink from 'components/ExternalLink';
 
-const TopicsLinks = props =>
-  props.topicCollections.length > 0 ? (
-    <ul className="coa-ThemesTopicsMenu__topics">
-      {props.topicCollections.map((tc, index) => (
-        <li
-          key={index}
-          className="coa-ThemesTopicsMenu__topic"
-          onKeyDown={props.handleFullSiteMenuItem}
-        >
-          <a
-            href={`/${props.intl.locale}${
-              tc.node.slug ? `/${props.themeSlug}/${tc.node.slug}` : tc.node.url
-            }`}
-            className="coa-ThemesTopicsMenu__link"
-          >
-            {tc.node.title}
-          </a>
-        </li>
-      ))}
-    </ul>
-  ) : (
-    <p className="coa-FullSiteMenu__coming-soon">
-      {props.intl.formatMessage(i18n2.comingSoon)}
-    </p>
-  );
+import ChevronRightBlue from 'components/SVGs/ChevronRightBlue';
+import WorkInProgressBanner from 'components/WorkInProgress/WorkInProgressBanner.js';
+
+const TopicsLinks = props => {
+
+  let columnWidth = ""
+  if (props.topicCollections.length > 4) {
+    columnWidth = "2col"
+    if (props.topicCollections.length > 8) {
+      columnWidth = "3col"
+    }
+  }
 
-const ThemesTopicsMenu = props => {
   return (
-    <nav
-      className={`coa-ThemesTopicsMenu ${
-        props.extraClasses ? props.extraClasses : ''
-      }`}
-    >
-      <ul className="coa-ThemesTopicsMenu__list">
-        {props.menu.map((theme, index) => (
-          <li className="coa-ThemesTopicsMenu__section" key={index}>
-            <h4
-              className="coa-ThemesTopicsMenu__theme"
-              tabIndex="0"
+    <div className="coa-ThemesTopicsMenu__topics-container">
+
+      {props.topicCollections.length > 0 ? (
+
+        <ul className={ "coa-ThemesTopicsMenu__topics coa-ThemesTopicsMenu__topics"+columnWidth }>
+          {props.topicCollections.map((tc, index) => (
+            <li
+              key={index}
+              className="coa-ThemesTopicsMenu__topic"
               onKeyDown={props.handleFullSiteMenuItem}
             >
-              {theme.text}
-            </h4>
-            <TopicsLinks
-              handleFullSiteMenuItem={props.handleFullSiteMenuItem}
-              topicCollections={theme.topicCollectionPages.edges}
-              themeSlug={theme.slug}
+              <a
+                href={`/${props.intl.locale}${
+                  tc.node.slug ? `/${props.themeSlug}/${tc.node.slug}` : tc.node.url
+                }`}
```

Comment:
Is this not being managed by Joplin instances/urls?

Reply:
@Niicck this was the current state of the theme links taken from line 16-18 above. The props only have the slug and the theme to construct the page link. Wasn't sure if that was intentional or not. 
- I'm guessing the page build builder and/or api queries may need to be tweaked?

So, i'm thinkin' i could investigate and fix that here. Or, create an issue out of it.

Reply:
If that's how it was before, we can leave it. If it ain't broke, let's leave it alone